src/main/archetype/core/pom.xml

```diff
@@ -57,24 +62,24 @@
     <dependencies>
         <!-- OSGi Dependencies -->
         <dependency>
-            <groupId>org.apache.felix</groupId>
-            <artifactId>org.apache.felix.scr</artifactId>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.core</artifactId>
```

Comment:
I am pretty sure the right artifactId is `osgi.core` (http://search.maven.org/#artifactdetails%7Corg.osgi%7Cosgi.core%7C6.0.0%7Cjar) and not `org.osgi.core`. Compare also with http://blog.osgi.org/2015/08/release-6-of-osgi-compendium-osgi.html.